test/demo/clearAll.inttest.ts

```diff
@@ -31,32 +31,18 @@ const notificationWithOnCloseActionResult2: NotificationOptions = {
     onClose: {task: 'close-2'}
 };
 
-type OuterTestParam = [
+type EnvironmentTestParam = [
     string,
-    () => void,
-    () => void
+    'center-open' | 'center-closed'
 ];
 
-type InnerTestParam = [
+type CallAllCallTestParam = [
```

Comment:
`CallAll` => `ClearAll`?.. And same for the constant below.

Reply:
✅